src/lib/chips/chip-list.ts

```diff
@@ -221,4 +292,14 @@ export class MdChipList implements AfterContentInit, OnDestroy {
     return index >= 0 && index < this.chips.length;
   }
 
+  /** Utility to check if an input element has no value. */
+  private static _isInputEmpty(element: HTMLElement): boolean {
```

Comment:
Rather than making this a private static method, just make it a top level function in the module (not exported)